fs/remote/resolver.go

```diff
@@ -413,6 +413,7 @@ func (f *httpFetcher) fetch(ctx context.Context, rs []region, retry bool) (multi
 	}
 	req.Header.Add("Range", fmt.Sprintf("bytes=%s", ranges[:len(ranges)-1]))
 	req.Header.Add("Accept-Encoding", "identity")
+	req.Header.Add("User-Agent", socihttp.UserAgent)
```

Comment:
Do we want to set this on all requests to the registry? I think we probably do.

We can defer that for now, but we should mention it in https://github.com/awslabs/soci-snapshotter/issues/759

Reply:
All span requests? I think so. 

> We can defer that for now, but we should mention it in https://github.com/awslabs/soci-snapshotter/issues/759

Sure, I can mention that.